app/src/androidTest/java/com/hedvig/app/feature/insurance/detail/CoverageTest.kt

```diff
@@ -55,9 +55,5 @@ class CoverageTest {
                 }
             }
         }
-
-        onScreen<ContractDetailScreen.CoverageTab.PerilBottomSheetScreen> {
```

Comment:
This code validated that the `PerilBottomSheet` was opened, can we keep that assertion in place? 🙂